BSU-SUITS-Team/ARSIS-6
# Move LMCC

# Description

This moves the [LMCC Repo](https://github.com/BSU-SUITS-Team/GCSiteTest) to a subfolder of this project

## Testing

To test:
- Download the repo
- Checkout this branch
- Initalize Submodules
  - `git submodule init`
  - `git submodule update --remote`
- Run `docker compose up --build -d` To build
- Verify that the LMCC is still reachable
